website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -288,26 +288,42 @@ Expert Mode Keyboard Shortcuts:
+### Installing Processing 4
 
-Go ahead and move it to your `Applications` folder, and launch the application. If this is the first time that you are running Processing, it will create what it calls it's `Sketch` folder in the following locations:
+First, go to Processing's website and [download the latest version of Processing](https://processing.org/download). For more information on Processing, you can review the official [Processing Getting Started Guide](https://processing.org/tutorials/gettingstarted) or the [Processing Documentation](https://processing.org/environment).
 
-On a Mac `Users/<user-name>/Documents/Processing`  
- On Windows `C:\Users\<user-name>\Documents\Processing`  
- On Linux `/home/<user-name>/sketchbook`
+Go ahead and move the Processing application to your `Applications` folder, and launch the application. If this is the first time that you are running Processing, it will create what it calls it's `Sketch` folder in the following locations:
 
-![GUI repo](../../assets/SoftwareImages/OpenBCISoftware/ganglion_GUI-repo.png)
+- On a Mac -`Users/<user-name>/Documents/Processing`  
+- On Windows - `C:\Users\<user-name>\Documents\Processing`  
+- On Linux -`/home/<user-name>/sketchbook`
 
-The OpenBCI GUI code repository is located on github [here](https://github.com/OpenBCI/OpenBCI_GUI). click on the `Clone or download` button in green on the right, and select `Download ZIP`. If you are a advanced GitHub user, go ahead and clone it, or fork it if you like.
+:::note
+Linux Users
+
+[Here is a walkthrough on how to install Processing on Ubuntu Linux.](http://www.artsnova.com/processing/installing-processing-ubuntu-linux-tutorial.html)
+:::
+
+:::note
+Mac Users
+
+Please use the __Intel 64-bit__ version of Processing 4 on all Mac computers. This will still run great on Apple Silicon. This allows us to continue using libraries that may not have a build for Arm64 architecture.
+:::
 
-If you downloaded the repository as a zip, please unzip it and you can change the folder name from `OpenBCI_GUI-master` to `OpenBCI_GUI`, if you prefer. You can then place this folder wherever you like (Mac Example: `Users/<user-name>/Documents/GitHub/OpenBCI_GUI`).
+### Downloading the code for the OpenBCI GUI
+
+The OpenBCI GUI code repository is located [here on Github](https://github.com/OpenBCI/OpenBCI_GUI). Complete the following steps to download it to your computer:
+
+- Click on the `Clone or download` button in green on the right, and select `Download ZIP`. If you are a advanced GitHub user, go ahead and clone it or fork it as your normally would.
+
+- If you downloaded the repository as a zip, please unzip it and you can change the folder name from `OpenBCI_GUI-master` to `OpenBCI_GUI`, if you prefer. You can then place this folder wherever you like (Mac Example: `Users/<user-name>/Documents/GitHub/OpenBCI_GUI`).
```

Comment:
This is a run on sentence. Perhaps:

Unzip the repository if you downloaded it as a zip. You can also optionally change the folder name from `OpenBCI_GUI-master` to `OpenBCI_GUI`.